### PR TITLE
glow 0.2.0 (new formula)

### PR DESCRIPTION
**Pull request by dawidd6**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

A CLI tool for rendering Markdown.

https://github.com/charmbracelet/glow

### CODE REVIEW

**Comment by issyl0** [comment]
The author already maintains a [tap](https://github.com/charmbracelet/homebrew-tap/blob/master/glow.rb) for this, so this source build PR might be of interest to them so that they can do less work. 😀

**Comment by dawidd6** [comment]
It's a goreleaser tap. Not really much work maintaining it, since it's updated automatically on release.

But yes. Would be wise to inform upstream author about this PR.

File: Formula/glow.rb [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    (testpath/"test.md").write <<~EOS
+      # header
+
+      **bold**
+
+      ```
+      code
+      ```
+    EOS
+    assert_match "# header", shell_output("#{bin}/#{name} test.md")
```
Comment (fxcoudert) [inline Formula/glow.rb]:
```suggestion
    assert_match "# header", shell_output("#{bin}/glow test.md")
```

**Review by fxcoudert — CHANGES_REQUESTED** [review]

**Review by issyl0 — APPROVED** [review]
Looks good, is notable enough, and passes all the CI checks. 💪 Thanks, @dawidd6!

**Pull request closed without merging**